libs/plugins/stream-client/src/lib/run-stream/run-stream.service.ts

```diff
@@ -64,7 +64,7 @@ export class RunStreamService {
       ]);
   }
 
-  close() {
+  closePopover() {
     this.overlayRef.dispose();
```

Comment:
is there a case where this can be called and `overlayRef` is already disposed? I think we should guard against it, example:
```typescript
closePopover() {
 if (this.overlayRef) {
   this.overlayRef.dispose();
   this.overlayRef = null;
 }
}
```

Reply:
i think there is no such case